ReactiveObjC/RACSubscriber.h

```diff
@@ -10,6 +10,8 @@
 
 @class RACCompoundDisposable;
 
+NS_ASSUME_NONNULL_BEGIN;
```

Comment:
Just a quick nit: the `NS_ASSUME_NONNULL_*` family of macros doesn't need a trailing semicolon in my experience.